va_explorer/templates/va_data_management/index.html

```diff
@@ -30,5 +32,19 @@
     {% endfor %}
   </tbody>
 </table>
+{% if va_data.has_other_pages %}
+  <nav aria-label="Page navigation conatiner"></nav>
+    <ul class="justify-content-center">{% bootstrap_pagination va_data size="large"%}</ul>
+    <ul class="pagination justify-content-center">
```

Comment:
Do we still need the custom pagination stuff here? Not sure if it's needed with the bootstrap_pagination or if it's duplicated?

Reply:
Forgot to add a comment about this. I left them in because you have to choose a specific page number with the bootstrap paginator, so I thought it could be nice to have the 'next' and 'previous' functionality once you've honed in on a page range. Can definitely remove if you think it's unnecessary though.